tools/go-cli/build.gradle

```diff
@@ -37,6 +38,19 @@ ant.condition(property: "arch", value: "386") {
     os(arch: "x86_32")
 }
 
+// Sets the ext.dockerBuildArgs from the whisk properties file that contains the build timestamp
+def setDockerBuildArgs() {
+    Properties props = new Properties()
+    String propsFileName = project.projectDir.getPath()+"/default.props"
+    props.load(new FileInputStream(propsFileName))
+    props.each { property ->
```

Comment:
Can't the whole loop be rewritten with one `props.find { key, value -> key == "WHISK_VERSION_DATE") }`?